common.mk

```diff
@@ -301,6 +308,10 @@ filelist-tee-common: optee-client xtest helloworld
 			"$(HELLOWORLD_PATH)/ta/8aaaf200-2450-11e4-abe2-0002a5d5c51b.ta" \
 			"444 0 0" 					>> $(fl); \
 	fi
+	@if [ "$(QEMU_USERNET_ENABLE)" = "y" ]; then \
```

Comment:
I thought that it would better to avoid "No eth0 interface found" message after each boot if network was intentionally disabled.
And in this case check in ` /etc/init.d/udhcpc` can be be useful in case if someone decide to manually run `/etc/init.d/udhcpc start`. But anyway, if it's critical I can just remove it, and add slink into `filelist.txt` in `gen_rootfs`